_includes/main_nav_tr.html

```diff
@@ -1,6 +1,13 @@
  <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top pl-md-5 pr-md-5 pl-sm-3 pr-0">
 
-    <a class="navbar-brand" href="{{ site.url }}/{{ page.lang }}/"><img src="{{ site.url }}/images/bisq-logo.svg" height="30"/></a>
+   <div>
+     <a class="navbar-brand" href="{{ site.url }}/{{ page.lang }}/"><img src="{{ site.url }}/images/bisq-logo.svg" height="30"/></a>
+
+     <!-- Outdated translation alert tool tag-->
+     {% if page.outdated_translation %}
+       <span class="navbar-text" style="color: orange;">Outdated translation</span>
```

Comment:
It looks like this is doing more than simply removing a variable from the downloads page?

Reply:
@huey735
huey735 Author Member Pending

@wiz I decided to put this commit in this PR because it also affects the downloads pages. It felt wrong to remove the 'version' variables in one PR and then add 'outdated_translations' in another PR when they overlap files.